tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/stomp/StompTest.java

```diff
@@ -2223,7 +2224,7 @@ public void testSameMessageHasDifferentMessageIdPerConsumer() throws Exception {
       ClientStompFrame frame2 = conn.receiveFrame();
       String secondMessageID = frame2.getHeader(Stomp.Headers.Message.MESSAGE_ID);
       assertNotNull(secondMessageID);
-      assertTrue(!firstMessageID.equals(secondMessageID), firstMessageID + " must not equal " + secondMessageID);
+      assertNotEquals(firstMessageID, secondMessageID, firstMessageID + " must not equal " + secondMessageID);
```

Comment:
The message is now just duplicating object value which the assert will cover itself (unlike prevously where it would just have said expected true). 

Could generalise it to say what was wrong, e.g "MessageIDs should not be equal"